ios/ItaliaApp/de.lproj/Localizable.strings

```diff
@@ -0,0 +1,2 @@
+"ALERT_DEVICE_ROOTED_TITLE" = "Device jailbroken";
```

Comment:
@dhinterlechner could you please help us with the translation of these 2 strings? 🙏